docs/Discussion.md

```diff
@@ -33,8 +19,16 @@ Other Features:
 
 1. Decent amortized runtime, but the first execution in a while is time consuming due to Lambda’s “probably start a new instance” feature
 
-## Challenges and solutions
+## Other Useful Services
+
+[Sagemaker](https://aws.amazon.com/cn/sagemaker/) is a stable AWS official tool to Build, train, and deploy machine learning models at scale. 
```

Comment:
like mentioned in other PR please use words from official documentation for all AWS Services that you talk about (including AWS SAM)

Reply:
Yep! All these words are from official website.